packages/mdctl-axon-tools/packageScripts/ingestTransform.js

```diff
@@ -23,6 +29,11 @@ module.exports = class extends Transform {
     if (!studySchemaCursor.hasNext()) return
 
     memo.studySchema = studySchemaCursor.next()
+    memo.c_sties = c_sites.find()
```

Comment:
typo in `c_sties`, btw, for thousands of records (in sites) will the memo work?

Reply:
Yea, you might be right. Fixing